demo/src/router.ts

```diff
@@ -20,6 +19,8 @@ export const examples = Object.entries(import.meta.glob<ModuleNamespace>("./exam
 const routes = {};
 
 examples.forEach((example) => {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
Because of `ModuleNamespace` is not available anymore, imports become of type `unknown`. So telling the compiler that everything's OK here